.konflux/scripts/subscription-manager/subscription-manager-bro.sh

```diff
@@ -12,8 +12,10 @@ SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
 SECRET_NAME_IN_KONFLUX="subscription-manager-activation-key"
 SECRET_KONFLUX_WORKSPACE_PATH="/workspace/${SECRET_NAME_IN_KONFLUX}"
+# The mount is provided by the buildah task when the ACTIVATION_KEY parameter is set to a valid secret name.
+SECRET_MOUNT_PATH="/activation-key"
 SECRET_KEY="activation-key"
-SECRET_LOCAL_PATH="${SCRIPT_DIR}/${SECRET_KEY}"
+SECRET_LOCAL_PATH="${SECRET_MOUNT_PATH}/${SECRET_KEY}"
 SECRET_INFO_URL='https://docs.engineering.redhat.com/pages/viewpage.action?pageId=407312060'
 
```

Comment:
Please check the item starting with `echo -n "2.` in `function usage`. This item is now outdated.

Moreover, I strongly suspect the `smuggle` command now is broken (which you can try if you undo all changes of this PR in `.tekton` directory). It tries to write the activation key to `/activation-key` directory which is not (and was not) persisted.

Either we should kill `smuggle` completely and replace item (2) of usage with instructions to use buildah param or fix `smuggle` so that the user of the script can decide whether to smuggle or to provide buildah param.

Since `smuggle` existed for tekton pipelines and buildah param should not hopefully go away, I think deleting `smuggle` should be fine.

Reply:
Done in https://github.com/stackrox/stackrox/commit/7002a83cdc445f2eb76b7b9b9a161f738d0544a3, https://github.com/stackrox/stackrox/commit/f91a0d43cfb2d774d484a6c012182f93faa93100